config.json

```diff
@@ -4,7 +4,7 @@
   "blurb": "",
   "exercises": [
     {
-      "slug": "hello-world",
+      "slug": "hello_world",
```

Comment:
Oh shit. I didn't realize that it was formatting all of the exercise slugs like this now.  That is definitely not okay.

Reply:
So, I know it is probably not ideal for us to use strings instead of symbols for the exercise slugs in the scheme configs, but can we do that?

Trying to differentiate between the slugs and the `UUID`s is going to be more trouble than we are avoiding with symbols for the slug names I think.

What is your take on this @jitwit  ?  Are you willing to compromise there?

Reply:
Yeah, I was about to comment here about that. I don't think it makes sense to put any of this logic inside the json library, so I'm thinking the best move is to preprocess the `track-config` in scheme, then send it to json for pretty printing. I already wrote it to filter out `wip` problems in preprocessing

Reply:
cool. where is that preprocessor?

Reply:
Oh, I see it is in the `process-config` function.

Reply:
I think it makes sense for this work to happen after we have filtered out the `wip`s.

I'll add a function to do this ;)

Reply:
Haha yeah. It's very modest at the moment. 

Ps. I think the `track-config` is so small that we don't need to worry about optimizing

Reply:
Yeah, that is why I figure another function is best even if it loops through the config another time.  I think it will be much easier to understand than adding the logic into the preprocessor function you have.

Reply:
Fixed